package.json

```diff
@@ -2,7 +2,7 @@
   "name": "web-vitals",
   "version": "0.2.1",
   "description": "Easily measure performance metrics in JavaScript",
-  "type": "module",
+  "type": "commonjs",
```

Comment:
Can you just remove the `type` config entirely?

I originally tried to make this whole package `"type": "module"` (with export maps and everything), but then a couple of the testing tools didn't work with the `.cjs` extension, so I removed the export map, but I guess forgot to remove the `"type": "module"` config.